config/content.json

```diff
@@ -24,6 +24,7 @@
             "/docs/metrics/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
             "/docs/cloud-identity/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-identity-2.0/",
             "/docs/cloud-monitoring/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-monitoring/",
+            "/docs/dedicated-load-balancers/v2/dev-guide/": "https://github.com/rackerlabs/docs-dedicated-networking/",
```

Comment:
I made it that way because it is a dedicated product, and also EA.  The doc is only visible to internal users now.  When it goes public, I thought we could change it.  

I also am open to the suggestion that this is not a good idea, and it should be developer-guide.